components/inputs/input-checkbox.js

```diff
@@ -19,6 +19,7 @@ export const checkboxStyles = css`
 		display: inline-block;
 		height: 1.2rem;
 		margin: 0;
+		outline: none;
```

Comment:
Noticed this when testing on Firefox:
![image](https://user-images.githubusercontent.com/13419300/120041912-21bf6f80-bfd7-11eb-94cc-a8ec22a02af1.png)